ceph_diagnostics_show/commands/osd

```
@@ -63,7 +63,11 @@ def osd_sum(ids, filter_dev_type):
     hosts = {}
     dev_types = {"hdd": 0, "hybrid": 0, "ssd": 0}
     for osd in m:
-        if not int(osd["rotational"]):
+        # Check for SSDs/NVMe through multiple indicators
+        if not int(osd["rotational"]) or \
+           (osd.get("bluestore_bdev_dev_node", "").startswith("/dev/nvme")) or \
+           (osd.get("bluestore_bdev_devices", "").startswith("nvme")) or \
+           osd.get("bluestore_bdev_type") == "ssd":
```

Hm. Interesting, could you provide a report example, when we need this? Because AFAIR some code in ceph was relying on "rotational" when selecting if to use _hdd or _ssd configuration parameters (see e.g. [1]), and in that case it might  be doing a wrong decision. I would check this, because we may want to output a warning in such cases instead of silently hide the issue. I recall there was an issue like this in the past when running ceph in VMs, and then it was  better to fix the VM (by setting the rotational bit via /proc/) to make ceph properly recognize what config params to use.

[1] https://github.com/ceph/ceph/blob/main/src/osd/OSD.cc#L3659

---

I'm double checking the report I was working with and this `osd sum` check seem to work as intended, apologies for the confusion. 

I was messing around with trying to add nvme into the check and commited some broken code.